app/ui.js

```diff
@@ -59,6 +59,11 @@ const UI = {
         // Translate the DOM
         l10n.translateDOM();
 
+        WebUtil.fetchJSON('../package.json')
```

Comment:
What happens if this fails? It's not terribly obvious that `package.json` is needed for the web pages, so people might be deploying without it.

Reply:
The idea solution imho would be to package noVNC before deployment (with rollup or such).
That way, the number just get injected here and there is no need to import the whole package.json file just for the number.
But you want the git repo to be deployable as is, so this is the only possibility I came up with.

I can add a catch case and simply say `Unkown` if package.json is not there.

Reply:
> I can add a catch case and simply say Unkown if package.json is not there.

Good enough for me